WordPress/src/main/res/layout/post_list_author_selection_dropdown.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:foreground="?attr/selectableItemBackground"
+              android:gravity="center"
+              android:orientation="horizontal"
+              android:paddingBottom="@dimen/margin_medium"
+              android:paddingEnd="@dimen/margin_medium"
+              android:paddingStart="0dp"
```

Comment:
Do we need to set the `paddingStart` to `0dp` manually?

Reply:
It is recommend by Google to always include a `paddingStart` when setting a `paddingEnd`. I believe this is to improve the clarity for future developers when they read it, and avoid change to underlying styles causing negative affects.

I mostly do it for the former.

Reply:
Makes sense, thanks for elaborating on this!